io.github.qalculate.Qalculate.yaml

```diff
@@ -0,0 +1,107 @@
+app-id: io.github.qalculate.Qalculate
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+command: qalculate-gtk
+rename-appdata-file: qalculate-gtk.appdata.xml
+rename-desktop-file: qalculate-gtk.desktop
+rename-icon: qalculate
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Network access
+  # for Update Exchange Rates
+  - --share=network
+  # Filesystem access
+  # Import CSV File
+  # Export CSV File
+  # Save Result Image
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-public-share
+  # Session Bus access
+  - --own-name=io.github.Qalculate
```

Comment:
It seems to use `io.github.Qalculate` as an app-id, any reason for not using that instead of `io.github.qalculate.Qalculate`? Such decision should be discussed with the upstream developers.

Reply:
The main reason io.github.qalculate.Qalculate is used instead of io.github.Qalculate is to avoid inconveniences for users of the Qalculate! flatpak package currently distributed (see https://github.com/flathub/flathub/pull/869). This might also make sense if flatpaks for separate parts (the library and the command line program) of the Qalculate project someday will be added.

I am the upstream developer.

Reply:
> The main reason io.github.qalculate.Qalculate is used instead of io.github.Qalculate is to avoid inconveniences for users of the Qalculate! flatpak package currently distributed (see #869).

I don't see how that is an issue? They have to install the one distributed by Flathub once available to continue receiving the updates.

>  This might also make sense if flatpaks for separate parts (the library and the command line program) of the Qalculate project someday will be added.

Flatpak isn't meant to distribute libraries but rather desktop applications (GUI & CLI)


> I am the upstream developer.

Perfect! I wasn't aware of that.

Reply:
> I don't see how that is an issue? They have to install the one distributed by Flathub once available to continue receiving the updates.

I was primarily thinking that users will loose application configuration and data (unless manually copied).

Reply:
I have now changed the app-id to io.github.Qalculate.

Reply:
> > I don't see how that is an issue? They have to install the one distributed by Flathub once available to continue receiving the updates.
> 
> I was primarily thinking that users will loose application configuration and data (unless manually copied).

That could be an issue indeed, but that's inevitable for now :/ so it's better to just have a proper app-id set now instead of having to fix it someday. Thanks!